ctapipe/reco/event_classifier.py

```diff
@@ -3,6 +3,8 @@
 from astropy import units as u
 
 from sklearn.ensemble import RandomForestClassifier
+from sklearn.neural_network import MLPClassifier
+from sklearn.preprocessing import StandardScaler
 
```

Comment:
So, if I get it correctly, you suggest to import the `MLPClassifier` and `StandardScaler` modules in the test file? Or don't care about those test at all?